sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,26 @@
+
+REPLACE INTO `spell_custom_attr` VALUES (63124, 131072); -- quest There's Something About the Squire (13654)
```

Comment:
TrinityCore does not use `REPLACE INTO`.
Please use `DELETE FROM` + `INSERT INTO` instead.